installer/PowerToysBootstrapper/bootstrapper/bootstrapper.cpp

```diff
@@ -138,6 +138,7 @@ int Bootstrapper(HINSTANCE hInstance)
         ("no_full_ui", "Use reduced UI for MSI")
         ("s,silent", "Suppress MSI UI and notifications")
```

Comment:
```suggestion
        ("s,silent", "Suppress MSI UI, notifications and does not start PowerToys")
```
Does it also suppress the .NET install UI?

Reply:
yes, it does